ultradns-rest/src/main/java/denominator/ultradns/model/Region.java

```diff
@@ -0,0 +1,238 @@
+package denominator.ultradns.model;
```

Comment:
I am not a big fan of these multiple nested if else blocks & neither do PMD, so please plan to refactor the code in this class in a better way. Use some design pattern to make it more elegant.